wiki/FitNesseRoot/HsacExamples/SlimTests/BrowserTests.wiki

```diff
@@ -56,6 +56,7 @@ The heuristic to find places tries a number of translations of end user terms to
 |scroll to <place>                                                         |scrolls the designated place into view                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                 |
 |hover over <place>                                                        |simulates hovering over the designated place with the mouse (so e.g. 'mouseover' behavior can be tested)                                                                                                                                                                                                                                                                                                                                                                                                                                                                                               |
 |drag and drop <place> to <target>                                         |drags the designated place to the target (does not work for HTML5 drag 'n' drop, because of [[Selenium issue][https://github.com/seleniumhq/selenium-google-code-issue-archive/issues/3604]])                                                                                                                                                                                                                                                                                                                                                                                                          |
+|click with offset <place> X <XinPixels> Y <YinPixels>                     |clicks with a offset X,Y (offset measured in pixels of Top Left corner of element)                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                     |
```

Comment:
In your test you actually use: `click <place> with offset X <horizontalOffset> Y < verticalalOffset > `
I have to say I like that better. 
Why is 'Top Left' capitalised?